Assets/HoloToolkit/Input/Scripts/InputEventData/SourcePositionEventData.cs

```diff
@@ -14,16 +14,18 @@ public class SourcePositionEventData : BaseInputEventData
         /// <summary>
         /// The new position of the source.
         /// </summary>
-        public Vector3 Position { get; private set; }
+        public Vector3 PointerPosition { get; private set; }
+        public Vector3 GripPosition { get; private set; }
 
         public SourcePositionEventData(EventSystem eventSystem) : base(eventSystem)
         {
         }
 
-        public void Initialize(IInputSource inputSource, uint sourceId, object tag, Vector3 position)
+        public void Initialize(IInputSource inputSource, uint sourceId, object tag, Vector3 pointerPosition, Vector3 gripPosition)
```

Comment:
I could leave a third Vector3 called position and default it to either one to prevent breaking, since this is event data. I think making position explicit as far as which one it's referring to is important and worth a breaking change.